lib/langchain/tool/weather.rb

```diff
@@ -30,6 +30,13 @@ class Weather < Base
         Action Input: Kiev, Ukraine; metric
     DESC
 
+    SCHEMA_PARAMETERS = {
+      input: {
+        type: "string",
+        description: "Comma separated city and unit (optional: imperial, metric, or standard)"
+      }
```

Comment:
now that supports multiple arguments, this can be better to split in 2, city and unit. with city required, and unit optional